BrowserStack.java

```diff
@@ -15,15 +14,17 @@
  *         isEmpty()    - return true if stack is not empty else false
  *
  */
+import java.util.*;
 public class BrowserStack {
 
-    String[] myStack;
+    ArrayList<String> myStack = null;
```

Comment:
Initially, I did only with String array.Since it was mentioned as dynamic growing stack, I used array list.